src/OrbitGl/ThreadTrack.cpp

```diff
@@ -386,8 +386,8 @@ constexpr float kMinimalWidthToHaveBorder = 4.0;
 [[nodiscard]] bool ThreadTrack::ShouldHaveBorder(
     const TimerInfo* timer, const std::optional<orbit_statistics::HistogramSelectionRange>& range,
     float width) const {
-  if (!range.has_value() || width < kMinimalWidthToHaveBorder ||
-      timer->function_id() != app_->GetHighlightedFunctionId()) {
+  if ((!range.has_value() || width < kMinimalWidthToHaveBorder || !app_->HasCaptureData()) ||
+      (app_->GetCaptureData().ProvideScopeId(*timer) != app_->GetHighlightedFunctionId())) {
```

Comment:
Please add a TODO here: This should refactored, UI classes should speak to CaptureData through the Provider.

Reply:
After an offline discussion, we agreed to ignore this.

Reply:
I will take care of create the bug :)